src/SelectDropdown.js

```diff
@@ -134,7 +134,7 @@ const SelectDropdown = (
   /* ******************** Render Methods ******************** */
   const renderSearchView = () => {
     return (
-      search && (
+      search && data.length && (
```

Comment:
Remove this line and keep 230 line codes would be more stable.
If keep 137 line, user can't do searching again when search results is empty :)

Reply:
I use `data.length`, not `dataArr.length`. And it's still possible for users search again.

Reply:
At the first time in this PR, I used `dataArr.length` at line 137, that might confuse you this time.